bin/README.md

```diff
@@ -0,0 +1,94 @@
+Customized interpreter for Alk
+==============================
+
+Requirements:
+-------------
+
+* The K framework - use the version of K indicated [here](https://github.com/alk-language/k-semantics).
+  It is highly recommended to put the paths to K's binaries in your system's PATH.
+
+How to use?
+-----------
+
+1. Compile the definition Alk that resides in the `alk` directory of this repo:
+
+```> kompile alk.k```
+
+or 
+
+```> pathToK/bin/kompile alk.k```
+
+This will generate a file called `alk-kompiled` in the same directory.
+
+2. Add `bin` (the folder where this README file resides) into your system's PATH.
+
+3. Run `alk` (or `alk.exe` on Windows) for the first time:
+
+```> alk```
+
+and you'll get:
+```
+Please provide an Alk file.
+Usage:
+alk(.exe) [-dhiks] [long options...] <file.alk>
+        -i STR --init STR         the initial Alk state
+        -s --stack                show stack content
+        -k STR --krun STR         path to krun
+        -d STR --directory STR    path to directory where alk-kompiled resides
+
+        -h --help                 print usage message and exit
+```
+
+The tool typically requires an Alk program, an initial state, and the directory containing `alk-kompiled`. 
+This is how you run programs (this assumes that `krun` is in your system's PATH and you are in the top-level directory of this repo):
+
+```
+> alk tests\miscelanea\gcd.alk --init "u |-> 42 v |-> 56" --directory alk
```

Comment:
"I meant that the running command should very simple."
I agree with that, but there are some things to discuss first.
* What is the default location of  `alk-kompiled` (recall that you said `alk gcd`, and this tells me that you want to execute `alk` *anywhere*)?
* I understand that you prefer `alk gcd` instead of  `alk gcd.alk`. Although, it's easy to do that in Perl, my personal opinion is that doesn't add any value to the tool; also, other tools like `gcc` or `java` do not have this feature. Anyway, I will not fight for this.